src/reducers/query.jsx

```diff
@@ -31,6 +31,7 @@ export const defaultQuery = {
   subLens: '',
   tab: types.MODE_MAP,
   totalPages: 0,
+  trend_depth: 5,
```

Comment:
🐫  case or 🐍  case?

Reply:
should be camel case for JS standards.  the api wants the 🐍  though